client/my-sites/stats/stats-detail-weeks/index.jsx

```diff
@@ -133,16 +132,14 @@ export default React.createClass( {
 						iconType = 'arrow-down';
 					}
 
-					cells.push(
-						<td className={ changeClass } key={ 'change' + index }>
-							<span className="value">
+					cells.push( <td key={ 'average' + index }>
+						{ this.numberFormat( week.average ) }
+							<span className={ 'value ' + changeClass } key={ 'change' + index }>
```

Comment:
A minor thing, but this `value` class is problematic since it doesn't match the component. We should use `stats-detail-weeks__value.is-rising`.